auditbeat/module/auditd/audit_linux_test.go

```diff
@@ -275,6 +275,58 @@ func TestMulticastClient(t *testing.T) {
 	assertHasBinCatExecve(t, events)
 }
 
+func TestRuleReload(t *testing.T) {
+	if !*audit {
+		t.Skip("-audit was not specified")
+	}
+
+	logp.TestingSetup()
+	FailIfAuditdIsRunning(t)
+
+	c := map[string]interface{}{
+		"module":      "auditd",
+		"socket_type": "unicast",
+		"reload": map[string]interface{}{
+			"period":  time.Second * 5,
+			"enabled": true,
+		},
+		"audit_rule_files": []string{
+			"/tmp/test.rules",
+		},
+	}
+
+	time.AfterFunc(time.Second, func() {
+		rule := fmt.Sprintf(`
+		   -a always,exit -F arch=b64 -F ppid=%d -S execve -k exec
+		`, os.Getpid())
+		err := ioutil.WriteFile("/tmp/test.rules", []byte(rule), 0644)
+		if err != nil {
+			t.Error(err)
+		}
+		logp.Info("writed rule file")
+	})
+
+	defer func() {
+		e := os.Remove("/tmp/test.rules")
+		if e != nil {
+			t.Error(e)
+		}
+	}()
+
+	time.AfterFunc(time.Second*7, func() {
+		exec.Command("cat", "/proc/self/status").Output()
+		logp.Info("run cat command")
+	})
+
+	logp.Info("start beat")
+	ms := mbtest.NewPushMetricSetV2(t, c)
+
+	events := mbtest.RunPushMetricSetV2(15*time.Second, 0, ms)
+	logp.Info("end beat")
+	assertNoErrors(t, events)
+	assertHasBinCatExecve(t, events)
+
+}
```

Comment:
pretty sure the lack of a newline here is going to result in a linter error, try making sure to run `mage fmt` in the `auditbeat` directory.

Reply:
Thanks spotting this. I ran mage fmt in the auditbeat directory, but did not get any prompt or correction about it. Corrected it anyway.

```
 ~/go/src/github.com/elastic/beats/auditbeat                                                                                                                [13:23:56] 
> $ mage fmt                                                                                                                                           [±feature-reload-auditd-rule-file ✓]
>> fmt - go-licenser: Adding missing headers
>> fmt - goimports: Formatting Go code
>> fmt - autopep8: Formatting Python code
```